kurtosis-devnet/cmd/main.go

```diff
@@ -165,6 +212,10 @@ func getFlags() []cli.Flag {
 			Name:  "environment",
 			Usage: "Path to JSON environment file output (optional)",
 		},
+		&cli.StringFlag{
+			Name:  "conductor-config",
```

Comment:
nit: `conductor-config-path`

Also, should we describe what the behaviour is when this flag is NOT provided?